### PR TITLE
my tynacoon order and small search input fix

### PR DESCRIPTION
**Pull request by agessler**
FIX FOR https://trello.com/c/EybhXEDf/77-fix-spacing-positioning-of-my-tynacoon-in-sidebar-navigation

### CODE REVIEW

**Comment by apepper** [comment]
Is the original workaround in https://github.com/Scrivito/scrivito-portal-app/blob/baf15c88f24bf3a06705fe753bc9279be0750e5d/src/Objs/SubnavigationOverview/SubnavigationOverviewLayoutComponent.tsx#L100-L101 still needed? If not, please remove it in this branch.

**Comment by apepper** [comment]
While reviewing this I noticed, that the "search box" is "right in the middle" of the mobile navigation:
Screenshot 2024-01-15 at 14 07 39

It is possible to move it to the very bottom of the list, even below the "Current User" menu?

**Review by apepper — COMMENTED** [review]
FEEDBACK ([f6e7766](https://github.com/Scrivito/scrivito-portal-app/pull/241/commits/f6e7766c7f11cf81ff6add6a4c5fbab6caa9747f))

Very nice 🚀!

Please see my comments for further discussions.

**Comment by agessler** [comment]
Search is now on top in mobile.

Bildschirmfoto 2024-01-15 um 14 43 55

File: src/assets/stylesheets/index.scss [hunk 1]
```diff
@@ -1201,16 +1201,21 @@ section:focus >,
...
   .navbar-meta,
   .navbar-main {
     flex-flow: column;
 
     .navbar-nav {
       margin: 0 !important;
     }
+    &.navbar-nav {
+      flex-flow: column-reverse;
+    }
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:1217]: Please try to position the search input at the very end of the list, or just above the "Current user" menu.

**Review by apepper — COMMENTED** [review]
FEEDBACK ([876ad47](https://github.com/Scrivito/scrivito-portal-app/pull/241/commits/876ad47dd4d8a067c971e3e5b184e1bb5d8ba448))

Already really nice 🚀. Only one smaller issue (see comments).

File: src/assets/stylesheets/index.scss (hunk 1 above)
Comment (apepper) [reply src/assets/stylesheets/index.scss]: Sorry, I didn't make it clear: The search input box should not be at the very end of _this_ list, but at the very end of navigation (main and meta combined). So either below "My Tynacoon" or (better) below "Current user info".

Comment (agessler) [reply src/assets/stylesheets/index.scss]: Possible with markup change.

Comment (apepper) [reply src/assets/stylesheets/index.scss]: Can you change the markup then?

File: src/assets/stylesheets/index.scss [hunk 2]
```diff
@@ -1198,10 +1198,24 @@ section:focus >,
   }
 }
 
+.navbar-main form {
+  display: block;
+}
+.navbar-meta form {
+  display: none;
+}
```
Comment (apepper) [inline src/assets/stylesheets/index.scss:1206]: This looks strange.

File: src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx [hunk 3]
```diff
@@ -64,6 +65,7 @@ export const MetaNavigation = connect(function MetaNavigation({
           <CurrentUserDropdown widget={widget} root={root} />
         </Nav>
       )}
+      <SearchBox searchResultsPage={root.get('siteSearchResultsPage')} />
```
Comment (apepper) [inline src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx:68]: You're now rendering the `SearchBox` twice and hiding one of them according to the current viewport? Is there maybe another solution to this?

**Review by apepper — COMMENTED** [review]
FEEDBACK ([c298c6f](https://github.com/Scrivito/scrivito-portal-app/pull/241/commits/c298c6fea6ae5736631850746978b2046e75410e))

I like how the search field is now "well positioned". But I'm not happy with the implementation (see my comments).

File: src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx (hunk 3 above)
Comment (agessler) [reply src/Widgets/NavigationWidget/SubComponents/MetaNavigation.tsx:68]: Sure.. we can introduce a real grid for whole navbar section and define grid areas. Everything custom - not part of Bootstrap.

File: src/assets/stylesheets/index.scss (hunk 2 above)
Comment (agessler) [reply src/assets/stylesheets/index.scss:1206]: No comment!

**Comment by apepper** [comment]
If I'm not mistaken most (if not all) is addressed in https://github.com/Scrivito/scrivito-portal-app/pull/400. Can this PR be closed, @agessler ?

**Comment by apepper** [comment]
Closing for now. We have to pick up this work later on.

**Pull request closed without merging**